lib/methods/veres-one-client.js

```diff
@@ -218,15 +215,15 @@ class VeresOneClient {
    * @returns response {Promise<Response>} from a fetch() POST.
```

Comment:
Is this accurate?

Reply:
Changed it to refer to an axios response. Thanks.